ndcube/ndcube.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 
 import abc
+import warnings
+from itertools import product
```

Comment:
Delete this import as it is no longer used.

Reply:
Oh yeah, I missed this one. I will delete it tomorrow.